gio/src/unix_fd_list.rs

```diff
@@ -42,27 +42,29 @@ pub trait UnixFDListExtManual: IsA<UnixFDList> + Sized {
     }
 
     #[doc(alias = "g_unix_fd_list_get")]
-    fn get(&self, index_: i32) -> Result<RawFd, glib::Error> {
+    fn get(&self, index_: i32) -> Result<OwnedFd, glib::Error> {
         unsafe {
             let mut error = ptr::null_mut();
-            let ret = ffi::g_unix_fd_list_get(self.as_ref().to_glib_none().0, index_, &mut error);
+            let raw_fd =
+                ffi::g_unix_fd_list_get(self.as_ref().to_glib_none().0, index_, &mut error);
             if error.is_null() {
-                Ok(ret)
+                let fd = OwnedFd::from_raw_fd(raw_fd);
+                Ok(fd)
             } else {
                 Err(from_glib_full(error))
             }
         }
     }
 
     #[doc(alias = "g_unix_fd_list_peek_fds")]
-    fn peek_fds(&self) -> Vec<RawFd> {
+    fn peek_fds(&self) -> &[BorrowedFd<'_>] {
```

Comment:
This must return a `Vec<RawFd>` too because this type here has inner mutability...